### DISCUSSION TITLE
New version v58 rev 1.49 : to sell

### DISCUSSION THREAD

**Discussion by heinemannj** [nliaudat/esp32_8ch_motor_shield#47] [Buy or Sell]
Today I've ordered  5 boards at jlcpcb.com.

I beleave three of them I don't  need and will be available for sell.

Estimated Timeline: 1 weeks

**Comment by dragop4** [comment]
I'm interested. Where are you from?

**Reply by heinemannj** [reply]
I'm living in Germany.

I have already received the boards.
Please contact me via private mail.
